commercial/app/AppLoader.scala

```diff
@@ -56,8 +50,10 @@ trait CommercialServices {
   lazy val feedsParser = wire[FeedsParser]
 }
 
-trait AppLifecycleComponents {
-  self: FrontendComponents with Controllers with CommercialServices =>
+trait AppComponents extends FrontendComponents with CommercialControllers with CommercialServices {
+
+  lazy val devAssetsController = wire[DevAssetsController]
+  lazy val healthCheck = wire[HealthCheck]
```

Comment:
@alexduf Not related to removing Play.current but as we mentioned earlier, simplifying AppLoader.
It makes sense to have traits for controllers and services since they are also used in Standalone app but lifecycleComponents, devAssetsController and healthCheck can be defined in AppComponents